src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -3742,10 +3741,10 @@ namespace LowTempRadiantSystem {
         Real64 ZeroFlowSurfTemp;  // Temperature of radiant surface when flow is zero
         int ZoneNum;              // Zone pointer for this radiant system
 
-        VarFlowRadDesignData variableFlowDesignDataObject{
-            state.dataLowTempRadSys->HydronicRadiantSysDesign(this->DesignObjectPtr)}; // Contains the data for variable flow hydronic systems
+        VarFlowRadDesignData variableFlowDesignDataObject = // Is this intended to be a deep copy?
```

Comment:
We're doing deep copies here. I didn't look closely to see if it was truly necessary.